xls/contrib/xlscc/unit_tests/expr_clone_test.cc

```diff
@@ -0,0 +1,393 @@
+// Copyright 2025 The XLS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "xls/contrib/xlscc/expr_clone.h"
+
+#include "clang/AST/Expr.h"
```

Comment:
I think other xlscc files uses this style of includes:
```
#include "clang/include/clang/AST/..."
```
See:
https://github.com/google/xls/blob/431afd7be8a8677def24bc79e24bd1f3a2dc86e5/xls/contrib/xlscc/cc_parser.cc#L37